modules/swagger-codegen/src/main/resources/go/README.mustache

```diff
@@ -38,25 +38,49 @@ Class | Method | HTTP request | Description
 {{/model}}{{/models}}
 
 ## Documentation For Authorization
-
-{{^authMethods}} All endpoints do not require authorization.
+{{^authMethods}} Endpoints do not require authorization.
 {{/authMethods}}{{#authMethods}}{{#last}} Authentication schemes defined for the API:{{/last}}{{/authMethods}}
-{{#authMethods}}## {{{name}}}
-
+{{#authMethods}}
+## {{{name}}}
 {{#isApiKey}}- **Type**: API key 
-- **API key parameter name**: {{{keyParamName}}}
-- **Location**: {{#isKeyInQuery}}URL query string{{/isKeyInQuery}}{{#isKeyInHeader}}HTTP header{{/isKeyInHeader}}
+!!! NOT IMPLEMENTED !!!
```

Comment:
That is correct. I could not see in the original where it was supported as a reference of how to implement it.

I could take another look at some other implementations and try to determine. Is there a way to write tests for it?